api/api/settings.py

```diff
@@ -90,6 +90,7 @@
     'EXCEPTION_HANDLER': 'desecapi.exception_handlers.handle_db_unavailable',
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.NamespaceVersioning',
     'ALLOWED_VERSIONS': ['v1', 'v2'],
+    'NON_FIELD_ERRORS_KEY': '*',
```

Comment:
Let's change this to something that would be a valid variable name, to avoid client issues